mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -10,6 +10,33 @@
 from mitosheet.state import State
 from mitosheet.transpiler.transpile_utils import TAB, column_header_to_transpiled_code
 
+# This is a helper function that generates the code for formatting the excel sheet
+def get_format_code(state: State) -> list:
+    code = []
+    formats = state.df_formats
+    for sheetIndex in range(len(formats)):
+        sheet_name = state.df_names[sheetIndex]
+        format = formats[sheetIndex]
+    
+        # If there is no formatting, we skip trying to access the colors
+        if format.get('headers') is None:
+            continue
+        
+        # Prepares the header font color and background color to be used in the API call
+        # So either they should be surrounded by quotes, or they should be None
+        header_font_color = None
+        header_background_color = None
+        if (format['headers'].get('color') is not None):
+            header_font_color = f'"{format["headers"]["color"]}"'
```

Comment:
We have a really badly named utility called `column_header_to_transpiled_code` that you can call on these values and it will put them in a string. So it would be like:

```
header_font_color = column_header_to_transpiled_code(format["headers"]["color"])
```